src/components/box/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+import { Box as ChakraBox, BoxProps as ChakraBoxProps } from '@chakra-ui/react';
+
+const Box: FC<ChakraBoxProps> = ({
+  children,
+  background,
+  borderRadius = 'sm',
+  color,
+  padding,
+  ...rest
+}) => {
+  return (
+    <ChakraBox
+      bg={background}
+      color={color}
+      borderRadius={borderRadius}
+      padding={padding}
+      {...rest}
+    >
+      {children}
+    </ChakraBox>
+  );
+};
+
+export default Box;
```

Comment:
```suggestion
export default Box;import { Box } from '@chakra-ui/react';
export default Box;
```

I would not redefine the props and map them, but just export the chakra box in this case.
- Defaulting the border-radius might be more of an edge case and we likely will use the box more often without background
- bg/background I have in mind chakra allows for both so we might wanna keep that behaviour

Reply:
Thank you for the suggestion! Let's keep it as generic as possible 👍